docs/REST/throttling.md

```diff
@@ -5,9 +5,13 @@ Configuration stores have limits on the requests that they may serve. Any reques
 Throttling is divided into different quota policies:
 
 **TotalRequests** - total amount of requests/sec
```

Comment:
NIT: 'sec' -> 'second' to be consistent

Reply:
I start a new pr in our repo, will fix those here first: https://msazure.visualstudio.com/Azure%20AppConfig/_git/AppConfigService/pullrequest/2668448?_a=files